src/index.html

```diff
@@ -1,13 +1,319 @@
+                <div class="category__gallery">
+
+                  <a href="#" class="category__link">
+                    <img
+                      src="./images/sh1.png"
+                      class="product__photo"
+                      alt="Smart Home 1"
+                    >
+                  </a>
+
+                  <a href="#" class="category__link">
+                    <img
+                      src="./images/sh2.png"
+                      class="product__photo"
+                      alt="Smart Home"
+                    >
+                  </a>
+
+                </div>
+
+                <h3 class="category__title">
+                  Smart Home
+                </h3>
+              </article>
+
+              <div class="category__block">
+                <a href="#" class="category__button section-button">View all</a>
+              </div>
+            </div>
+          </section>
+        </div>
+
+        <section id="about-us" class="about-us">
+          <div class="about-us__background"></div>
+          <div class="about-us__information">
+            <h2 class="section-title about-us__title">Timeless, for 50 years and counting</h2>
+
+            <p class="about-us__text">
+              Bang & Olufsen is expanding its Recreated Classics Program with the launch of Beosystems – a limited edition music system that transcends time by bridging the gap between one of our iconic designs from 1972 and today’s cutting-edge digital technology.
+            </p>
+          </div>
+        </section>
+      </div>
+    </main>
+
+    <footer>
+      <section id="contact-us" class="contact-us">
+        <div class="container">
+          <h2 class="section-title">Contact us</h2>
+          <div class="contact-us__content">
+            <form action="#" class="contact-us__form form">
+
+              <label for="name">
+                <input
+                  id="name"
+                  type="text"
+                  class="form__input"
+                  placeholder="Name"
+                >
+              </label>
+
+              <label for="email">
+                <input
+                  id="email"
+                  type="email"
+                  class="form__input"
+                  placeholder="Email"
+                >
+              </label>
+
+              <label for="message">
+                <textarea
+                  id="message"
+                  placeholder="Message"
+                  rows="6"
+                  class="form__input form__message"
+                ></textarea>
+              </label>
+
+              <button class="form__button section-button">
+                Send
+              </button>
+            </form>
+
+            <div class="contact-us__information contacts">
+              <div class="contacts__phone">
+                <p class="contacts__title">Phone</p>
+
+                <p class="contacts__text">+1 234 5555-55-55</p>
```

Comment:
5. [CHECKLIST] Make sure phone icons and phone numbers are real link to start a call